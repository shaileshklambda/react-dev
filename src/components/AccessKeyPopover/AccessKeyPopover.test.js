import React from "react";
import renderer from "react-test-renderer";
import Accesskeypopover from ".";

it("Accesskeypopover: default", () => {
  const component = renderer.create(<Accesskeypopover />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
