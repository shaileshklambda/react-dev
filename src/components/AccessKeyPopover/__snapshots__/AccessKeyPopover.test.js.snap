// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Accesskeypopover: default 1`] = `
<div>
  <img
    className=""
    height={20}
    onClick={[Function]}
    src={
      Object {
        "ReactComponent": [Function],
        "__esModule": true,
        "default": "access-key.svg",
      }
    }
    width={25}
  />
</div>
`;
